### PR TITLE
Adds robot-update instructions and troubleshooting for wifi/ssh

### PR DESCRIPTION
**Pull request by KaijenHsiao**
Several people are having issues with connecting to their robots because of problems with updating.  This PR adds instructions for how to update a new robot, as well as suggestions for troubleshooting wifi and ssh connection issues.

This PR references the Making and using a debug cable page that is in this PR:
https://github.com/KuriRobot/Kuri-Documentation/pull/19

### CODE REVIEW

File: guides/getting-started/connecting-to-kuri.md [hunk 1]
```diff
@@ -31,19 +31,72 @@ Through bluetooth the phone looks for Kuri and, if more than one is found, will
...
+after the current one).  
+
 ## Finding Kuri's hostname
 Kuri's hostname is based upon the serial number of the robot which is printed on the box as well as on the bottom of the robot.  The hostname is built with a base `kuri-0000` with the addition of the last numbers and letters from the serial number.
 
 For example, if Kuri's serial number is `KR118160000abc`, the hostname is `kuri-0000abc`.
 
+The exception is some (but not all) robots that start with 'KR01744', which had a different
+serial number scheme (don't ask).  If you have one such, take the last numbers from the
+serial number and convert that number to hexadecimal.  So, for serial number `KR017440000149`,
```
Comment (cjwilliams) [inline guides/getting-started/connecting-to-kuri.md]: If they were reprovisioned properly, I think the hostname should just be the serial. @po1, could you verify?

Comment (tisho) [reply guides/getting-started/connecting-to-kuri.md]: Definitely not true for any of the robots I've QA'd. Easiest way to figure out the hostname is to grab a phone, go to Settings > Bluetooth and look for anything that starts with `kuri-`. That name is the hostname of the robot.

Comment (po1) [reply guides/getting-started/connecting-to-kuri.md]: The HFUs will have a `kuri-000xx` hex host name with `xx` varying loosely from 30 to bf

Later units and all reprovisioned robots will have `kuri-0000xxx` as a hostname, with the same digits as the last 7 digits of their S/N

Comment (po1) [reply guides/getting-started/connecting-to-kuri.md]: Most HFUs that are out there were not reprovisioned

**Pull request merged**